docs/conflicts.md

```diff
@@ -47,3 +47,62 @@ The deeper understanding of conflicts has many advantages:
 
 For information about how conflicts are handled in the working copy, see
 [here](working-copy.md#conflicts).
+
+
+## Conflict markers
+
+Conflicts are "materialized" using *conflict markers* in various contexts. For
+example, when you run `jj edit` on a commit with a conflict, it will be
+materialized in the working copy. Conflicts are also materialized when they are
+part of a diff output (e.g. `jj show` on a commit that introduces or resolves a
+conflict). Jujutsu uses a different conflict marker style than Git uses. Here's
+an example:
+
+```
+<<<<<<<
+-------
++++++++
+ apple
+-grape
++grapefruit
+ orange
++++++++
+APPLE
+GRAPE
+ORANGE
+>>>>>>>
+```
+
+That represents a conflict where one side changed "grape" to "grapefruit" and
+the other side has "banana", "grape", and "orange". To resolve the conflict,
+you should keep the side that's preceded by just `+++++++` and apply the diff
```

Comment:
Maybe it would be clearer to just refer to the "first" and "second" sections, rather than specifying the markers?